### ISSUE TITLE
Questions about datasets

### ISSUE THREAD

**Issue by zhangyuge666** [WangYuhan-0520/Amazon-KG-v2.0-dataset#1]
Hello, I would like to ask whether the source of the movie data set is Amazon's data set in 2018 or 2023. Because I downloaded the data set in 2018, I can't find the corresponding item in the original data set by using the item name in your data, and user can only find part of it.

**Comment by WangYuhan-0520** [comment]
Hello! We used Amazon Review Data (2018) from https://nijianmo.github.io/amazon/index.html and linked the dataset's items to the DBpedia Knowledge Graph. It is important to note that some items cannot find corresponding KG entities in DBpedia, and we only keep the items with aligned entities and their related interactions in this dataset, so the dataset is inevitably downsized. 
Thank you for your attention!

**Comment by zhangyuge666** [comment]
Thank you for your reply. I have solved the problem. I want to know whether your paper has been pre-printed or officially published, because my research work uses a data set published by you, and I want to quote your paper in the introduction part of my paper data set.

**Comment by WangYuhan-0520** [comment]
Yes, this work has been accepted for publication by SIGIR2024. The title of the paper is "Amazon-KG: A Knowledge Graph Enhanced Cross-Domain Recommendation Dataset" and you can get information about it by visiting this page (https://sigir-2024. github.io/program_papers.html#rr). 
Thanks for the support!
